godot-core/src/property.rs

```diff
@@ -357,18 +365,27 @@ mod export_impls {
         (@export $Ty:ty) => {
             impl Export for $Ty {
                 fn default_export_info() -> PropertyHintInfo {
-                    PropertyHintInfo::with_hint_none(<$Ty as $crate::builtin::meta::GodotType>::godot_type_name())
+                    if sys::GdextBuild::since_api("4.3") {
+                        PropertyHintInfo::with_hint_none()
+                    } else {
+                        PropertyHintInfo::with_hint_none_type_name(<$Ty as $crate::builtin::meta::GodotType>::godot_type_name())
+                    }
                 }
             }
         };
 
         (@type_string_hint $Ty:ty) => {
-            impl TypeStringHint for $Ty {
+                impl TypeStringHint for $Ty {
```

Comment:
This indentation is probably a mistake.

Reply:
yeah, rustfmt isn't the best at formatting macros